docs/source/release/v5.1.0/muon.rst

```diff
@@ -41,6 +41,8 @@ Improvements
 - Improved the maintainability of the fitting code in the Muon Analysis interface by removing redundant and duplicated code.
   These changes will make the addition of new functionality in the future easier.
 - The plot guess option within the fitting tab will now update when a parameter is changed.
+- Updated :ref:`DoublePulseFit <algm-DoublePulseFit>` to allow composite function input.
```

Comment:
is this one supposed to be here? Can we also have a line saying TF asym and double pulse is now available in the GUI?

Reply:
Yes this change is also in this PR. I have added a line abut the GUI.